source/dlangbot/utils.d

```diff
@@ -2,8 +2,74 @@ module dlangbot.utils;
 
 import dlangbot.app : runAsync;
 
+import vibe.http.client : HTTPClientRequest, HTTPClientResponse;
+import vibe.http.common : HTTPStatus;
+
 import std.datetime : Duration;
 
+version (unittest) int _expectedStatusCode;
+
+bool expectedStatusCode(int statusCode) nothrow @safe @nogc
+{
+    version (unittest)
+    {
+        if (_expectedStatusCode == statusCode)
+        {
+            _expectedStatusCode = 0;
+            return true;
+        }
+    }
+    return false;
+}
```

Comment:
Hmm shouldn't the test logic be separated from the application logic?

Reply:
Yes, if you have a better idea, go for it. Doesn't really fit the `runAsync`, `runTrello` pattern.